scripts/draw_singlepad.py

```
@@ -0,0 +1,223 @@
+# Col 6 +5V extension (rail -> R4 right)
+wire((6, 1), (6, 22), RED, w=4)
+
+# Horizontal jumpers
+wire((3, 3), (4, 3), BLU, w=4)        # BUS to R1 lower
+wire((3, 16), (5, 16), ORN, w=4)      # J5.SIG to SBUS_IN ext
+wire((3, 18), (4, 18), PUR, w=4)      # J6.SIG to SBUS_OUT ext
+
+# -------- COMPONENTS --------
+
+def res_v(c, r1, r2, ref_val):
+    x1, y1 = xy(c, r1); x2, y2 = xy(c, r2)
```

Ruff (0.15.17)

[error] Multiple statements on one line (semicolon)

(E702)